src/sth_app.js

```diff
@@ -0,0 +1,41 @@
+/* globals process, require */
+
+(function() {
```

Comment:
Well, this is kind of a very common hack in Javascript to be able to use the 'use strict'; directive (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) in a controlled and limited context.

The syntax:
```javascript
(function() {
   .
})();
```
is a so called Immediately-Invoked Function Expression (IIFE) (see http://en.wikipedia.org/wiki/Immediately-invoked_function_expression) and it allows us to take advantage of the function scope Javascript provides. In Javascript there's only the global and the function scope (no block scope, for example). Well, I am talking about ECMAScript 5 (the current version of Javascript). In Javascript 6 a new keyword (let) will let us declare variables with a block scope :)

With this technique, we assure that the 'use strict'; directive is only applied to the contents of the function, this is the code of the file.
If we didn't use IIFE and our app used (imported) other Javascript files not requiring the 'use strict' mod, the 'use strict'; directive would be applied to them and errors may arise.